search-sentiments-map/public/script.js

```diff
@@ -75,3 +75,23 @@ function showMoreOrLess() {
     showMoreOrLessToggleItem.innerText = TOGGLE_SHOW_MORE;
   }
 }
+
+/** Gets the sentiment score of an inputted search topic. 
+ *  TODO(ntarn): Get the average sentiment score for the inputted search topic's search results. 
+ */
+function getSentiment() {
+  const searchTopic = document.getElementById('search-topic').value;
+  const sentimentScoreElement = document.getElementById('sentiment-score');
+  const searchTopicObject = "searchTopic="+encodeURIComponent(searchTopic);
+  fetch('/sentiment', {method: 'POST',  // Send a request to the URL.
+    headers: new Headers({
+      'Content-Type': 'application/x-www-form-urlencoded',
+    }),
+    body: searchTopicObject // Send data from index.html.
```

Comment:
You might want to specify that you're sending the search topic, since this is specific. It in't general formatted data as it usually would be. If that makes sense?